test/fixtures/index.test.ts

```diff
@@ -84,6 +84,20 @@ describe('buildDepGraph', () => {
       expect(isEqual).toBe(true);
     });
   });
+
+  it('on fixture depWithUnderscore yields graph and successfully resolves dependency with hash', () => {
```

Comment:
What does having a fixture for this behaviour add? 
Isn't this behaviour tested in the new unit test added?

Reply:
@razzmadaz I definitely felt like the fixture and the unit test were basically just repetitions of each other and felt conflicted about it.